lib/bulk_data/cache.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "bulk_data"
```

Comment:
Otherwise Rails won't load the parent BulkData module and treat the `module BulkData` that is defined in this class - missing the loading of lib/bulk_data.rb